### ISSUE TITLE
Handle Qualifying Elements (HTML)

### ISSUE THREAD

**Issue by TheJaredWilcurt** [red-perfume/red-perfume-html#6]
```css
h1.example {
  display: block;
  text-align: center;
}
```
should become
```css
h1.rp__display__--COLONblock {
  display: block;
}
h1.rp__text-align__--COLONcenter {
  text-align: center;
}
```
or if uglified
```css
h1.rp__0 {
  display: block;
}
h1.rp__1 {
  text-align: center;
}
```

**Comment by TheJaredWilcurt** [comment]
Work already done in this repo. Waiting on this to be added into `red-perfume-css`.
